test/new-e2e/system-probe/config/vmconfig-arm64.json

```diff
@@ -1,7 +1,95 @@
 {
     "vmsets": [
         {
-            "name": "distro_arm64",
+            "name": "only_tracer_distro_arm64",
+            "recipe": "distro-arm64",
+            "arch": "arm64",
+            "console_type": "file",
+            "kernels": [
```

Comment:
Should we consider a way to define the kernels separate for their inclusion in sets? It seems brittle to fully define them multiple times.

Reply:
Yes, I am working on a adding a task for generating the `vmconfig.json` file at runtime.